### PR TITLE
Announcements 2021 07 23

### PR DESCRIPTION
**Pull request by cokiengchiara**
No changes for apps. 

- [x] I'm checking w @snizhana-dynnyk  in Slack if she wants changes to Dashboards to be included.
- [x] Not sure if 'prepare helm values for configuration management' is worth announcing to customers? It's prep work for sth else? https://github.com/giantswarm/docs/blob/5661548cfb4c2736dc0b4a56f6aa6e936ecc6cff/src/content/changes/web-ui/happa/v1.14.0.md Maybe @snizhana-dynnyk , you know? Kuba made changes but he seems to be on vacation

### CODE REVIEW

**Comment by snizhana-dynnyk** [comment]
config management is an internal feature, we don’t announce it

**Comment by snizhana-dynnyk** [comment]
and there are  no new not-announced changes in Dashboards

**Comment by snizhana-dynnyk** [comment]
There is a new dashboard for KVM added by Rocket, I guess. Should it be included in the announcement if it is for KVM only?

**Comment by cokiengchiara** [comment]
> There is a new dashboard for KVM added by Fer ~Rocket, I guess~. Should it be included in the announcement if it is for KVM only?

That's true... Not sure, depends on how you'd like to generally treat changes to dashboards... Are KVM dashboards changes announced with KVM announcements?

**Comment by snizhana-dynnyk** [comment]
> Are KVM dashboards changes announced with KVM announcements?

I don’t think so. Maybe let's include it this time and I will put a card for the next SIG Product sync? Wdyt?

**Comment by cokiengchiara** [comment]
> > Are KVM dashboards changes announced with KVM announcements?
> 
> I don’t think so. Maybe let's include it this time and I will put a card for the next SIG Product sync? Wdyt?

Sounds good. Can you suggest the text 🙏 ?

File: src/content/changes/highlights/2021-07-23.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## User interfaces
+
+
+## Documentation
+
+Our [CRD docs](https://docs.giantswarm.io/ui-api/management-api/crd/) now feature deprecation info. On the list there is a DEPRECATED tag. On details pages, there is a box with details. In cases where the CRD is directly replaced by another one, this info is given in the box.
```
Comment (snizhana-dynnyk) [inline src/content/changes/highlights/2021-07-23.md:15]:
```suggestion
Our [CRD docs](https://docs.giantswarm.io/ui-api/management-api/crd/) now feature deprecation info. On the list, there is a DEPRECATED tag. On the details pages, there is a box with details. In cases where the CRD is directly replaced by another one, this info is given in the box.
## Dashboards

The KVM resource usage dashboard has been added to Grafana.
```

**Review by snizhana-dynnyk — CHANGES_REQUESTED** [review]

**Review by snizhana-dynnyk — APPROVED** [review]
lgtm

**Pull request merged**